sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -86,5 +297,30 @@ public async Task RecognizeEntitiesTypesSubTypes()
             Assert.AreEqual(NamedEntityType.Quantity, entities[14].Type);
             Assert.AreEqual(NamedEntitySubType.Temperature, entities[14].SubType);
         }
+
+        [Test]
+        public async Task RotateSubscriptionKey()
+        {
+            string endpoint = Environment.GetEnvironmentVariable("TEXT_ANALYTICS_ENDPOINT");
+            string subscriptionKey = Environment.GetEnvironmentVariable("TEXT_ANALYTICS_SUBSCRIPTION_KEY");
+
+            // Instantiate a client that will be used to call the service.
+            var credential = new TextAnalyticsSubscriptionKeyCredential(subscriptionKey);
+            var client = new TextAnalyticsClient(new Uri(endpoint), credential);
```

Comment:
Can't you use `GetClient` here as well?

Reply:
Good point. I need to figure out how to make it work with the recording though :|

Reply:
done